source/ShipyardPanel.cpp

```diff
@@ -101,6 +101,18 @@ void ShipyardPanel::DrawItem(const string &name, const Point &point)
 		return;
 
 	DrawShip(*ship, point, ship == selectedShip);
+
+	// Don't show the "in stock" amount if the ship is sold.
```

Comment:
This comment was correct, just maybe should be reworded a bit. And this is exactly the same way it works for outfits: "in stock" can be shown only if the shop doesn't sell this item normally.
```suggestion
	// Don't show the "in stock" amount if the shipyard has an unlimited stock of this ship.
```